components/core/Component/Overlay/OverlayTrigger.razor.cs

```diff
@@ -479,8 +479,13 @@ protected virtual async Task OnTriggerContextmenu(MouseEventArgs args)
             }
         }
 
-        protected virtual void OnMouseUp(JsonElement element)
+        /// <summary>
+        /// on document mouse up
+        /// </summary>
+        /// <param name="element"></param>
+        protected virtual async void OnMouseUp(JsonElement element)
         {
+            await Task.Delay(4);
```

Comment:
Why is it 4ms?

Reply:
In JS, 4ms is a better configuration, and I think the same goes for Blazor.

https://html.spec.whatwg.org/multipage/timers-and-user-prompts.html#dom-settimeout